src/main/resources/webpublic/index.js

```diff
@@ -23,27 +23,51 @@ function showSummary(container, summary) {
 /* Create a box with answer sources. */
 function showSources(container, sources) {
 	container.empty();
-	sources.forEach(function(s) {
+        $.each(sources, function(sid, source) {
 		var state_stags = ['<i>', '<b>', ''];
 		var state_etags = ['</i>', '</b>', ''];
-		container.append('<p class="source">'
-				+ '<img src="/wikipedia-w-logo.png" alt="W" class="wlogo" />'
-				+ ' <a href="http://en.wikipedia.org/?curid='+s.pageId+'" target="_blank">'
-				+ state_stags[s.state] + s.title + state_etags[s.state]
-				+ '</a> (' + s.origin + ')</p>'); // TODO also include the first sentence?
+        if(!(typeof (source.pageId) ==="undefined")) { //this forces to only show en wiki
```

Comment:
Hmm, do we have any better way to distinguish enwiki? Like some sort of "origin" or "type" field? We should!